cmd/slinky/main.go

```diff
@@ -176,6 +179,13 @@ func init() {
 		"",
 		"Use a custom listen-to endpoint for market-map (overwrites what is provided in oracle-config).",
 	)
+	rootCmd.Flags().BoolVarP(
+		&useCMCOnly,
+		"use-cmc-only",
+		"",
+		false,
+		"Use CoinMarketCap only for price data. **This should only be used for testing.**. Only works if you pair this with the --market-config-path flag.",
+	)
```

Comment:
I really think this is a bad idea and despite the usage string, someone will use this incorrectly. The best way to avoid misuse is to put this in the config imo.

Reply:
we could generate a CMC MM for every MM that we generate or separately run a script that can do the parsing logic.

Reply:
Im assuming former probably isn't ideal

Reply:
just FYI, we have several other fields in here where validators could incorrectly run things, is there a reason why you think this one is any different?

Reply:
We could also try to obfuscate with env variables.

Reply:
I'm moving the logic into a separate script.